### ISSUE TITLE
From in object issue

### ISSUE THREAD

**Issue by sanusi87** [SparkPost/nodemailer-sparkpost-transport#10]
I was able to send email using this fields in **Node.js**:

``` js
{
    to: 'xxx',
    from: 'xxx',
    subject: 'xxx',
    html: 'xxx'
}
```

but then i change the **from** field into and object:

``` js
{
    to: 'xxx',
    from: {
        name: 'xxx',
        address:'xxx'
    },
    subject: 'xxx',
    html: 'xxx'
}
```

i got his error:

``` js
{
    [SparkPostError: Unprocessable Entity]
    name: 'SparkPostError',
    errors:[
        {
            message: 'required field is missing',
            description: 'content.from.email is a required field',
            code: '1400' 
        } 
    ],
    statusCode: 422 
}
```

is from in object type not yet supported?

**Comment by simison** [comment]
Object model makes it so much easier to handle security, that it should definitely be added.

**Comment by aydrian** [comment]
Closed by #12
